modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -62,7 +62,7 @@ class ONNXImporter
 public:
 
     ONNXImporter(Net& net, const char *onnxFile)
-        : dstNet(net)
+        : dstNet(net), dispatch(buildDispatchMap())
```

Comment:
We need PR to master branch with merge of these changes and "diagnostic run" mode.